jekyll/_includes/quick-info-list.html

```diff
@@ -5,6 +5,7 @@
   <div id='arrow' data-popper-arrow></div>
 </div>
 <div class="quick-info-list">
+  {% if page.readTime != false %}
```

Comment:
Super nit, but I feel that `!= false` is a bit tricky to read. Is it possible to just use `{% if page.readTime %}` in liquid? And if not ... just use `{% if page.readTime == true %}`?

Reply:
Yup great point!

Reply:
I was following how we do page.toc I can update no worries! I agree makes sense